examples/next/visual-tests/react/demo/package.json

```diff
@@ -19,7 +19,7 @@
     "test": "node spec/support/jasmine.config.js"
   },
   "dependencies": {
-    "@handsontable/react": "latest",
+    "@handsontable/react": "14.0.0-next-b24d538-20231108",
```

Comment:
Could we go back to having `"latest"` as the version number in the `next`'s examples? 
This is a little nitpicky, as it _shouldn't_ break anything, but having `"latest"` as the placeholder version looks better than a build generated from a random commit.

A quick search shows me that there are `48` places where a version like this as assigned to either `handsontable` or the wrappers.

Reply:
At the moment, I'm fairly certain that this would break if we updated to the latest version, which is 13.1.0. The a11y demos, however, require version 14.0.0. The goal is to switch to either the next version or 14.0.0 upon release.

Reply:
The `next` examples should be linked to the local builds (both locally and on the CI), so _technically_ changing this should not break anything.
If it does, there might be something wrong with our examples scripts.